WordPress/Classes/Utility/BuildInformation/Device.swift

```diff
@@ -0,0 +1,40 @@
+/// Runtime information about the current device.
+///
+@objc
+class Device: NSObject {
```

Comment:
I'd expect an object called `Device` to know about iPad vs iPhone vs Watch, and be surprised to find it about simulator vs physical. Any thoughts on renaming this to something like `Runtime` or `DeviceRuntime`?

Reply:
Considering my task after wrapping this up will be to completely eliminate the need for this, `DeviceRuntime` sounds about right to me. :)

Sounds good?

Reply:
@diegoreymendez sounds great imo!